### PR TITLE
moss: Add verify command

### PR DESCRIPTION
**Pull request by tarkah**
Resolves https://github.com/serpent-os/moss/issues/126

### TODO
- [x] Cleanup CLI output

### CODE REVIEW

File: moss/src/cli/state.rs [hunk 1]
```diff
@@ -45,6 +45,11 @@ pub fn command() -> Command {
                     .value_parser(clap::value_parser!(u64)),
             ),
         )
+        .subcommand(
+            Command::new("verify")
+                .about("Verify TODO")
+                .arg(arg!(--verbose "Vebose output").action(ArgAction::SetTrue)),
+        )
```
Comment (tarkah) [inline moss/src/cli/state.rs:52]: We should probably add this as a global / root CLI option w/ short as `v` and change `version` short to `V`.

@ermo thoughts?

Comment (ermo) [reply moss/src/cli/state.rs:52]: @tarkah Yes! I was thinking the same thing last night actually.

**Comment by ermo** [comment]
@tarkah Needs a rebase ^^'

**Review by ermo — APPROVED** [review]
I'd rather have this in than not so we can begin testing it. If it has issues, we can fix/tweak later.

**Pull request merged**